theme-data/light/light-common.json

```diff
@@ -203,6 +203,14 @@
     "overlay": {
       "meetings": {
         "normal": "@color-white-alpha-70"
+      },
+      "audiodump": {
+        "primary": {
```

Comment:
the theme-data colours should only be ones defined by ux. these have already been defined for example color-gray-10 is defined as "@theme-background-solid-secondary-normal" we should be using existing colours to build components from. We should not be adding to this file